templates/page.html

```diff
@@ -347,12 +347,12 @@ <h3>Developers</h3>
         <div class="release-news">
           <img src="{{ '/static/img/release.png'|asseturl }}" />
           <div class="release-news-body">
-            <h3>REANA 0.9.2</h3>
-            <h6>19 Dec 2023</h6>
-            <p>Increase workflow restart limit.</p>
-            <p>Automatically mount any CVMFS repository.</p>
-            <p>Deploy REANA clusters on ARM architecture hardware.</p>
-            <p class="release-news-link"><a href="https://github.com/reanahub/reana/releases/tag/0.9.2">See full release notes</a></p>
+            <h3>REANA 0.9.3</h3>
+            <h6>13 Mar 2024</h6>
+            <p>Upgrade Snakemake to v7.</p>
+            <p>Improve job submission performance.</p>
```

Comment:
One more thing: could you perhaps include the screenshot window borders (as we have them on `blog.reana.io`) so that the illustrating image would look like a screenshot to avoid giving impression that these are just two composed individual images put together?